### DISCUSSION TITLE
Editable field to stack to save place

### DISCUSSION THREAD

**Discussion by webteamseo** [Laravel-Backpack/community-forum#845] [General]
Hi, is there a way to stack editable fields in one column?
Turn this:
![CleanShot 2024-02-21 at 14 22 55@2x](https://github.com/Laravel-Backpack/community-forum/assets/5486565/49135d88-e52b-4243-9126-1be2300a246e)

To something like this:

<img width="249" alt="CleanShot 2024-02-21 at 14 22 55@2x2" src="https://github.com/Laravel-Backpack/community-forum/assets/5486565/4c7769a2-d4cd-43af-8fe2-2e6479d1fcf8">

**Comment by karandatwani92** [comment]
Hey @webteamseo 

It seems you are on list operation & talking about [editable-columns ](https://backpackforlaravel.com/products/editable-columns).

No! It's not possible to put them horizontally.

**Comment by webteamseo** [comment]
thanks!
